src/renderer/stores/locations.js

```diff
@@ -39,31 +47,14 @@ ipcRenderer.on('DC_EVENT_LOCATION_CHANGED', (evt, payload) => {
 })
 
 // sometimes a MSGS_CHANGED is sent instead of locations changed
-ipcRenderer.on('DC_EVENT_MSGS_CHANGED', (evt, payload) => {
-  const { selectedChat, mapSettings } = locationStore.getState()
-  const { chatId } = payload
-  if (selectedChat && chatId === selectedChat.id) {
-    getLocations(chatId, mapSettings)
-  }
-})
-
-ipcRenderer.on('DC_EVENT_INCOMING_MSG', (evt, payload) => {
-  const { selectedChat, mapSettings } = locationStore.getState()
-  const { chatId } = payload
-  if (selectedChat && chatId === selectedChat.id) {
-    getLocations(chatId, mapSettings)
-  }
-})
-
-ipcRenderer.on('DD_EVENT_LOCATIONS_UPDATED', (evt, payload) => {
-  const { locations } = payload
-  locationStore.setState({ ...locationStore.getState(), locations: locations })
-})
+ipcRenderer.on('DC_EVENT_MSGS_CHANGED', onLocationChange)
+ipcRenderer.on('DC_EVENT_INCOMING_MSG', onLocationChange)
+ipcRenderer.on('DD_EVENT_LOCATIONS_UPDATED', onLocationChange)
```

Comment:
This seems a bit odd. The callback doesn't look the same for this event and is `DD_EVENT_LOCATIONS_UPDATED` correct?

Reply:
Nevermind, I thought `DD_EVENT_..` was an event from core.

Reply:
You were right - it was the wrong callback for that event. DD_EVENT means it's an event triggered by DeltachatDesktop